MerchantApi/Lib/Model/ShopperStatistics.php

```diff
@@ -174,6 +165,7 @@ public function __construct(array $data = null)
         $this->container['last_login'] = isset($data['last_login']) ? $data['last_login'] : null;
         $this->container['has_previous_purchases'] = isset($data['has_previous_purchases']) ? $data['has_previous_purchases'] : null;
         $this->container['fraud_check_result'] = isset($data['fraud_check_result']) ? $data['fraud_check_result'] : null;
+        $this->_currencyUtil = new \Zip\ZipPayment\MerchantApi\Lib\Model\CurrencyUtil;
```

Comment:
Hi @mahbub-zip 
You could also make the function inside CurrencyUtil to be public static function so that in all the code you would have:
use Zip\ZipPayment\MerchantApi\Lib\Model\CurrencyUtil;
CurrencyUtil::isValidCurrency($this->container['currency']);

which will be much easier than initializing the class